osh/hub/templates/waiving/result.html

```diff
@@ -32,6 +32,7 @@ <h1>{{ compare_title }}</h1>
         {% if sb.scan.tag %}
             <span id="scan_release" class="info_box black_font light_grey_bg margin_right">{{ sb.scan.tag.release.product }} release {{ sb.scan.tag.release.release }}</span>
         {% endif %}
+        <span class="info_box black_font margin_right">{{ scan_type }}</span>
```

Comment:
I would show the scan type only if it is not `Regular` to avoid confusing users in the most common case.

Reply:
Addressed in 9c2211cd78a6e7a067c873b62d247911cd74fd22.